core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/worker/DataTransferSupport.scala

```diff
@@ -13,7 +13,7 @@ import scala.collection.mutable
 import scala.concurrent.ExecutionContext
 import scala.util.control.Breaks
 
-trait DataTransferSupport extends BreakpointSupport {
+class DataTransferSupport(val sender: ActorRef) extends BreakpointSupport {
```

Comment:
Later we should decouple BreakpointSupport and DataTransferSupport. They shouldn't have an inheritance relationship